cleaners/firefox.xml

```diff
@@ -18,51 +18,133 @@
+    <!--
+    <value os="windows">Moonchild Productions\Pale Moon</value>
+    <value os="windows">Mozilla\SeaMonkey</value>
+    <value os="windows">Waterfox</value>
+    -->
+    <!-- Linux $$Browser$$ must be figured out for each browser, too! Tobias. -->
+    <value os="linux">mozilla/firefox</value>
+    <!-- If you already wanna give the universal cleaner a try: Remove the comment-marks from the following block... -->
+    <!-- The following values are just guesses... -->
+    <!--
+    <value os="linux">seamonkey</value>
+    <value os="linux">waterfox</value>
+    -->
+  </var>
   <var name="base">
-    <value os="windows">%AppData%\Mozilla\Firefox</value>
-    <value os="linux">~/.mozilla/firefox</value>
-    <value os="freebsd">~/.mozilla/firefox</value>
-    <value os="openbsd">~/.mozilla/firefox</value>
+    <value os="windows">%AppData%\$$Browser$$</value>
+    <value os="linux">~/.$$Browser$$</value>
+    <value os="freebsd">~/.$$Browser$$</value>
+    <value os="openbsd">~/.$$Browser$$</value>
   </var>
   <var name="profile">
-    <value search="glob" os="windows">%AppData%\Mozilla\Firefox\Profiles\*</value>
-    <value search="glob" os="linux">~/.mozilla/firefox/*</value>
-    <value search="glob" os="freebsd">~/.mozilla/firefox/*</value>
-    <value search="glob" os="openbsd">~/.mozilla/firefox/*</value>
+    <value search="glob" os="windows">$$base$$\Profiles\*</value>
+    <value search="glob" os="linux">~/.$$Browser$$/*</value>
+    <value search="glob" os="freebsd">~/.$$Browser$$/*</value>
+    <value search="glob" os="openbsd">~/.$$Browser$$/*</value>
+  </var>
+  <var name="localprofile">
+    <value search="glob" os="windows">%LocalAppData%\$$Browser$$\Profiles\*</value>
+    <!-- Unknow by other OSes, yet! Tobias B. Besemer will do Linux Part, later! -->
+    <!--
+    <value search="glob" os="linux">~/.$$Browser$$/*</value>
+    <value search="glob" os="freebsd">~/.$$Browser$$/*</value>
+    <value search="glob" os="openbsd">~/.$$Browser$$/*</value>
+    -->
   </var>
   <option id="backup">
     <label>Backup files</label>
     <description>Delete the backup files</description>
+    <!-- Why not delete them all?
     <action command="delete" search="glob" path="$$profile$$/bookmarkbackups/*.json"/>
     <action command="delete" search="glob" path="$$profile$$/bookmarkbackups/*.jsonlz4"/>
+    -->
+    <action command="delete" search="walk.files" path="$$profile$$/bookmarkbackups/"/>
+    <!-- Done! -->
+    <action command="delete" search="walk.files" path="$$profile$$\sessionstore-backups\"/>
+    <action command="delete" search="file" path="$$profile$$\sessionstore-backups\"/>
+    <action command="delete" search="glob" path="$$profile$$/sessionstore.bak-*"/>
+    <action command="delete" search="glob" path="$$profile$$\*.bak"/>
+    <action command="delete" search="glob" path="$$profile$$\*.bak-*"/>
+    <action command="delete" search="glob" path="$$profile$$\prefs-*.js"/>
```

Comment:
I don't have any of these (lines 121-124) on my system. Are these derived browsers like Pale Moon?

Reply:
No, my profile is just **really** old! ^^
But if they don't exist, they can't be deleted and no data can't be lost! ^^
So it's code to "cleanup" old profiles like mine... ^^

Reply:
Think I forgot to note the file date to this files like I later did...
Think the files are now already deleted on my system... But I can check for them...